community-concepts/productaanvraag/productaanvraag-denhaag.json

```
@@ -0,0 +1,89 @@
+{
+    "title": "Productaanvraag tbv terugbelnotitie Den Haag",
```

```suggestion
    "title": "Productaanvraag tbv terugbelnotitie Den Haag",
```

Wellicht deze iets generieker maken?